tests/integration/request_test.py

```diff
@@ -58,6 +58,18 @@ def test_send_specified_auth(mock_client):
     ).session == 'attacker_session'
 
 
+def test_str_encodes_array_path_parameters(mock_client):
+    request = FuzzingRequest(
+        operation_id='get_expect_path_array',
+        tag='types',
+        ids=[1, 2, 3],
+    )
+    request.send()
+    assert str(request) == (
+        'curl -X GET http://localhost:5000/types/path_array/1%2C2%2C3'
```

Comment:
Should we use the `URL` constant like the next test?

Reply:
👍 I'll also fix it for the test above too.